modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
+
+Instance scopes::
+Modes that determine how you can want to operate an Argo CD instance. The available modes are cluster-scoped instance and namespace-scoped instance.
+
+Namespace-scoped instance (Application delivery instance)::
+A mode wherein Argo CD is configured to only manage resources in certain namespaces on a cluster and use them for application delivery.
+
+Cluster-scoped instance::
+A mode wherein Argo CD is configured to manage every single resource on the cluster including certain cluster-specific resources such as cluster configuration, cluster RBAC, Operator resources, platform Operators, or secrets.
+
+Default Argo CD instance (Default cluster-scoped instance)::
+A default instance that a {gitops-title} Operator instantiates immediately after its installation, in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources.
+
+User-defined Argo CD instance::
+A custom Argo CD instance that you an install and deploy to manage cluster configurations or deploy applications. By default, any new user-defined instance has permissions to manage resources only in the namespace where it is deployed.
+
+ApplicationSet (Argo CD application set)::
+A resource object and a CRD that automatically generates Argo CD applications based on the contents of an `ApplicationSet` CR. Cluster administrators use this CRD to define a single `ApplicationSet` CR to generate and update multiple corresponding Argo CD `Application` CRs.
+
+Argo CD API server::
+A gRPC/REST server that exposes the API consumed by the Web UI, CLI, continuous integration (CI), and continuous deployment (CD) systems. 
+
+Ardo CD server (Argo CD-server)::
+A server that serves the API and UI for Argo CD.
+
+Argo CD resource (`ArgoCD` CR)::
+A CRD that describes the desired state for a given Argo CD instance that allows you to configure the components and settings which make up that Argo CD instance.
+
+Application Controller (Argo CD Application Controller)::
+A controller that continuously watches the Git repo for changes, monitors running applications, compares the current, live state against the desired target state, and deploys new changes. Examples include Argo CD Application Controller  detecting OutOfSync application state and optionally taking corrective action.
+
+Argo CD repository server (Argo CD-repo-server)::
+An Argo CD component that reads from Git repositories, generates corresponding application manifests, executes custom configuration management tooling, and returns the result to the Argo CD Application Controller. 
+
+ApplicationSet Controller (Argo CD ApplicationSet Controller)::
+A custom Kubernetes controller that exists within Argo CD and adds support for a custom `ApplicationSet` CRD.  This controller automatically generates Argo CD Applications based on the contents of an `ApplicationSet` CR and manages them, across multiple clusters declaratively, from a single mono-repository.
```

Comment:
Also I was not sure about the commas around "across multiple clusters declaratively ". Is that supposed to be an inserted sentence? Because it did not seem like that to me.